docs/source/models/causes/opioid_use_disorder/index.rst

```diff
@@ -0,0 +1,821 @@
+
+GBD 2023 Modeling Strategy
++++++++++++++++++++++++++++
+
+GBD 2023 Definition and Diagnostic Criteria
+--------------------------------------------
+
+The Global Burden of Disease (GBD) 2023 study defines opioid use disorders as "a maladaptive pattern of opioid abuse, leading to clinically significant impairment or distress that includes symptoms of dependence, such as withdrawal symptoms or progressive tolerance." [GBD-2023-Overview]_ The GBD disease modeling includes cases that meet the diagnostic criteria for opioid dependence as defined by the Diagnostic and Statistical Manual of Mental Disorders, 4th edition, text revision (DSM-IV-TR) and the International Classification of Diseases (ICD-9 and ICD-10).
+
+
+**DSM-IV-TR Diagnostic Criteria**: To meet the DSM-IV-TR criteria for opioid dependence, at least **three of the following seven symptoms** must be experienced within the same 12-month period:
+
+1. **Tolerance**, characterized by either:
+
+   - A need for increased amounts of the substance to achieve intoxication; or
+   - Markedly diminished effect with continued use of the same amount of the substance
+
+2. **Withdrawal**, characterized by either:
+
+   - Withdrawal symptoms characteristic to dependence; or
+   - The same (or similar) substance is taken to avoid withdrawal symptoms
+
+3. Substance taken in progressively larger amounts or for longer period
+4. Persistent desire or unsuccessful efforts to reduce substance use
+5. Disproportionate time dedicated to obtaining the substance
+6. Other important activities are given up because of the substance use
+7. Substance use is continued despite knowledge of physical or psychological problems occurring as a result of the substance
+
+GBD 2023 Non-Fatal Modeling Strategy
+-------------------------------------
+
+The GBD 2023 study uses DisMod-MR 2.1, a Bayesian meta-regression tool, to estimate the epidemiology of opioid use disorder. [DisMod-Methods]_ DisMod-MR 2.1 integrates diverse data sources to produce internally consistent estimates of key epidemiological parameters.
+
+**Key Epidemiological Parameters**
+
+DisMod-MR 2.1 estimates the following parameters for opioid use disorder:
+
+1. **Prevalence**: The proportion of the population with OUD at a given time
+2. **Incidence**: The rate at which new cases of OUD develop in the population
+3. **Remission**: The rate at which individuals with OUD transition to a non-disorder state
+4. **Excess mortality**: The elevated mortality risk associated with OUD beyond background mortality
+
+These parameters are estimated by age, sex, location, and year (1990-2023) and are constrained to be internally consistent through the DisMod-MR 2.1 modeling framework.
+
+**DisMod Prior Settings**
+
+Key modeling assumptions in GBD 2023 include:
```

Comment:
This overlaps (and somewhat contradicts? though incidence and YLLs/YLDs are different) the table below on restrictions.